Sensus.Android.Tools/Concurrent/MainConcurrent.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright 2014 The Rector & Visitors of the University of Virginia
```

Comment:
Might the various Tools projects be better as shared projects rather than class libraries?